core/templates/pages/license-page/license-page.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="oppia-license-page">
+  <div class="oppia-static-header">
+    <h1 [innerHTML]="'I18N_LICENSE_TERMS_HEADING' | translate"></h1>
+  </div>
+  <background-banner></background-banner>
+  <div>
+    <div class="oppia-static-content-below-banner">
+      <div class="oppia-page-card oppia-static-content">
+        <div class="about-tab-container">
```

Comment:
where is the style for these classes defined?

Reply:
removed